pkg/cmd/cli/cmd/delete.go

```diff
@@ -0,0 +1,103 @@
+package cmd
```

Comment:
I have opened an issue upstream and hope to get something out of it so I can submit a PR there.